docs/badges/quickstart.rst

```diff
@@ -0,0 +1,76 @@
+Quick Start
+===========
+
+    Learners *earn* badges based on Open edX platform activity.
```

Comment:
Should we expand this with a sentence-or-two about what a badge is? Might be useful for people that dive right into the `Quick Start` docs.